cookbook/asset/custom_version_strategy.rst

```diff
@@ -0,0 +1,205 @@
+.. index::
+   single: Asset; Custom Version Strategy
+
+How to use a Custom Version Strategy for Assets
+===============================================
+
+Out of the box, Symfony exposes configuration options which allow you to
+specify the assets version and the format of the versioned path. If you require
+more control, you need to create a custom version strategy.
+
+.. versionadded:: 2.7
+    The Asset component was introduced in Symfony 2.7.
+
+Default Package
+---------------
+
+The default package is the `''` package. It is used when you do not specify
+a package name in the :ref:`asset <reference-twig-function-asset>` Twig function.
+In order to override the version strategy used by the default package,
+it is necessary to add a compiler pass.
+
+This example shows how to integrate with `gulp-buster`_.
+
+Create Compiler Pass
```

Comment:
imo there should be a short info what the compiler pass is exactly used for in this case